lib/solana.cpp

```diff
@@ -533,9 +550,44 @@ Version Connection::getVersion() const {
 }
 
 uint64_t Connection::getFirstAvailableBlock() const {
-  // create request
   json params = {};
   const json reqJson = jsonRequest("getFirstAvailableBlock", params);
+
+  return sendJsonRpcRequest(reqJson);
+}
+
+uint64_t Connection::getSlot(const GetSlotConfig &config) const {
+  const json params = {config};
+  const json reqJson = jsonRequest("getSlot", params);
+
+  return sendJsonRpcRequest(reqJson);
+}
+
+uint64_t Connection::getminimumLedgerSlot() const {
+  const json params = {};
+
+  const json reqJson = jsonRequest("minimumLedgerSlot", params);
```

Comment:
actually should then be named `minimumLedgerSlot`

Reply:
renamed to "minimumLedgerSlot"